BraveRewardsUI/Common/Button.swift

```diff
@@ -80,7 +86,33 @@ class Button: UIButton {
             loaderView.removeFromSuperview()
           }
         }
-        break
+      case .system:
+        let isLoading = self.isLoading
+        let colour = self.titleColor(for: .normal)
```

Comment:
Won't this be clear and the image below be nil upon any subsequent loading toggles?